docs/modules/commons/pages/cli.adoc

```diff
@@ -0,0 +1,294 @@
+`build.gradle` test project file:
+
+.Enable treating of known issues as positive result for stories runs at project level
+[source,gradle]
+----
+runStories.treatKnownIssuesOnlyAsPassed = true
+----
+
+== Debug stories
+
+The task runs test stories omitting the vividus initialization check based on current xref:ROOT:tests-configuration.adoc[tests configuration]
+and tests state in the file system.
+
+.macOS / Linux:
+[source,shell]
+----
+./gradlew debugStories
+----
+
+.Windows:
+[source,shell]
+----
+gradlew debugStories
+----
+
+By default known issues occurred during stories debug fail the test execution result with corresponding xref:ROOT:tests-configuration.adoc#_exit_codes[exit code],
+in order to not consider known issues as failures use the following property:
+
+.Enable treating of known issues as positive result for current debug run
+[source,shell]
+----
+./gradlew debugStories -PtreatKnownIssuesOnlyAsPassed=true
+----
+
+The treating of known issues as passed can be enabled globally for project by adding the following line into the `build.gradle`
+test project file:
+
+.Enable treating of known issues as positive result for debug runs at project level
+[source,gradle]
+----
+debugStories.treatKnownIssuesOnlyAsPassed = true
+----
+
+== Print available steps
+
+The task prints in alphabetical order all the steps that are available for the configured profiles, environments and suites.
+
+Please see xref:ROOT:tests-configuration.adoc[Tests Configuration] for more details.
+
+.Options
+[cols="3,1,1,1", options="header"]
+|===
+
+|Description
+|Short notation
+|Long notation
+|Default value
+
+|Path to a file to save the list of available steps
+|f
+|file
+|by default steps are printed into the stdout
+
+|===
+
+.macOS / Linux:
+[source,shell]
+----
+./gradlew printSteps --args='-f my-steps.txt'
+----
+
+.Windows:
+[source,shell]
+----
+gradlew printSteps --args='-f my-steps.txt'
+----
+
+.Output
+[source,gherkin]
+----
+vividus                                             Then `$value` matches `$regex`
+vividus-plugin-web-app                              When I switch back to the page
+vividus-plugin-web-app                              When I press $keys on keyboard
+                            COMPOSITE IN STEPS FILE Then an element with the name '$elementName' exists
+                            COMPOSITE IN STEPS FILE When I click on an image with the name '$imageName'
+
+----
+
+== Count steps
+
+The task counts steps in the specified tests folder and prints them in the descending order.
+
+.Options
+[cols="3,1,1,1", options="header"]
+|===
+
+|Description
+|Short notation
+|Long notation
+|Default value
+
+|Directory to count steps
+|d
+|dir
+|story
+
+|Number of steps to print
+|t
+|top
+|<no limits by default>
+
+|===
+
+.macOS / Linux:
+[source,shell]
+----
+./gradlew countSteps --args='-t 5 -d story/uat'
+----
+
+.Windows:
+[source,shell]
+----
+gradlew debugStories --args='-t 5 -d story/uat'
+----
+
+.Output
```

Comment:
```suggestion
.Top used steps
```

Reply:
It's clear from the context so prefer to keep it as it is

Reply:
Example 1. Output
Example 2. Output
Example 3. Output

Will look ugly